docs/conf.py

```diff
@@ -13,14 +13,11 @@
 import docutils
 import os
 import re
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
-
 
 # -- Project information -----------------------------------------------------
 
 project = 'OpenROAD'
-copyright = 'The OpenROAD Project, 2020'
+copyright = 'The OpenROAD Project, 2021'
 author = 'OpenROAD Team'
 
 
```

Comment:
Is this the agreed format for the Copyright message

Reply:
I am not sure, but this is just for the readthedocs website (see bottom right corner [here](https://openroad.readthedocs.io/en/latest/)).
![image](https://user-images.githubusercontent.com/9001905/127366221-020fe446-0765-4c55-bee9-978556c3372d.png)

Reply:
No @abk-openroad has pointed out we should use "The Regents of the University of California" as "the openroad project" is not a formal entity than can own/assign a copyright.

Reply:
Fixed on 944e82c.